### PR TITLE
Minor bug fixes

### PR DESCRIPTION
**Pull request by SPPearce**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -4,9 +4,15 @@ editor_options:
     wrap: sentence
 ---
 
-# mesa 0.99.0
+# dev
 
-### Added
+### BUG FIXES
+* `summariseDMRsByGene` now correctly summarises genes when they have different positions or lengths in different windows. 
+* `plotCorrelationMatrix` no longer crashes no annotation is being used.
```
Comment (SPPearce) [inline NEWS.md:11]:
```suggestion
* `plotCorrelationMatrix` no longer crashes when no annotation is being used. 
```

**Comment by Steven-M-Hill** [comment]
version from 0.99.0 to 0.5.1? What's happening there?

**Comment by Steven-M-Hill** [comment]
> version from 0.99.0 to 0.5.1? What's happening there?

0.99.0 is the version number needed for BioConductor release, will revert to this just prior to this.

**Comment by SPPearce** [comment]
> version from 0.99.0 to 0.5.1? What's happening there?

I had put 0.99.0 because X.99.0 is required for the first submission on Bioconductor, but I want to get a version released here so that we can refer to it internally now.

**Pull request merged**